src/nvim/edit.c

```diff
@@ -4183,7 +4183,7 @@ static int ins_compl_get_exp(pos_T *ini)
                 if (tmp_ptr - ptr >= IOSIZE - len) {
                   tmp_ptr = ptr + IOSIZE - len - 1;
                 }
-                STRNCPY(IObuff + len, ptr, tmp_ptr - ptr);
+                snprintf((char *)(IObuff + len), tmp_ptr - ptr, "%s", ptr);
```

Comment:
This triggers the ASAN failure, and is not in the Vim patch.  Why was it done?

Reply:
IDK it was giving a linting error, maybe I should've run `make clean` before `make lint`